pyscriptjs/src/pyodide.ts

```diff
@@ -78,12 +78,15 @@ export class PyodideInterpreter extends Interpreter {
             await this.loadPackage('micropip');
         }
         logger.info('pyodide loaded and initialized');
-        this.run('print("Python initialization complete")')
+        await this.run('print("Python initialization complete")')
     }
 
-    run(code: string): unknown {
-        return this.interface.runPython(code);
+    /* eslint-disable */
+    async run(code: string): Promise<{result: any}> {
+        // TODO: ADD A COMMENT TO EXPLAIN WHY RESULT IS WRAPPED IN A DICTIONARY
+        return { result: this.interface.runPython(code) };
```

Comment:
yes please, add a comment :)
Writing it here so that we don't forget :)

Reply:
done!